docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.7'
+
+services:
+    libmedium:
+        image: realaravinth/libmedium
+        container_name: 'libmedium'
+        hostname: 'libmedium'
+        user: nobody
+        #read_only: true
+        restart: always
+        security_opt:
+        - no-new-privileges:true
+        cap_drop:
+        - ALL
+        cap_add:
+        - CHOWN
+        - SETGID
+        - SETUID
+        ports:
+            - '8082:7000'
+        volumes:
+            - './config/default.toml:/etc/libmedium/config.toml'
```

Comment:
Docker mounts directories to directories, so this line will try to mount `./config/default.toml/` to `/etc/libmedium/config.toml`, which won't work:

```
19:38 atm@lab tmp.PIc6Mq9Z4A → docker-compose up -d
[+] Running 0/1
 ⠿ Container libmedium  Starting                                                                        0.5s
Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/tmp/tmp.PIc6Mq9Z4A/config/default.toml" to rootfs at "/etc/libmedium/config.toml": mount /tmp/tmp.PIc6Mq9Z4A/config/default.toml:/etc/libmedium/config.toml (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
```

I know `./config/default.toml` is an inconvenient name (since it will require renaming) but I think here you can assume that the admin will rename their file before deploying libmedium.